doc/api/cli.md

```diff
@@ -411,14 +411,18 @@ Node options that are allowed are:
 - `--enable-fips`
 - `--force-fips`
 - `--icu-data-dir`
+- `--inspect-brk`
```

Comment:
Seems useful to have, although if the node process spawns a child will that child also try to connect to the inspector?